docs/source/ops/requirements.rst

```diff
@@ -26,3 +26,8 @@ Daml is tested using the following specific dependencies in default installation
 2. `Eclipse Adoptium <https://adoptium.net>`_ version 11 for Java.
 3. PostgreSQL 10.0
 4. Oracle Database 19.11
+
+Feature/Component System Requirements
+-------------------------------------
+
+1. :doc:`The JavaScript Client Libraries <../app-dev/bindings-ts/index>` is tested on Node 14.18.3. A version greater or equal to 14.18.3 is recommended. 
```

Comment:
```suggestion
1. :doc:`The JavaScript Client Libraries <../app-dev/bindings-ts/index>` are tested on Node 14.18.3. with typescript compiler 4.5.4. Versions greater or equal to these are recommended. 
```

Reply:
@stefanobaghino-da I'd stick with this. Telling people a working set of tools is of value.